src/pages/waiting-room/__tests__/waiting-room.spec.ts

```diff
@@ -6,28 +6,35 @@ import { AppModule } from '../../../app/app.module';
 import { WaitingRoomPage } from '../waiting-room';
 import { AuthenticationProvider } from '../../../providers/authentication/authentication';
 import { AuthenticationProviderMock } from '../../../providers/authentication/__mocks__/authentication.mock';
-import { Store } from '@ngrx/store';
+import { Store, StoreModule } from '@ngrx/store';
 import { StoreModel } from '../../../shared/models/store.model';
 import { By } from '@angular/platform-browser';
 import {
   ToggleResidencyDeclaration,
   ToggleInsuranceDeclaration,
 } from '../../../modules/test/pre-test-declarations/pre-test-declarations.actions';
 import { PreTestDeclarationsModule } from '../../../modules/test/pre-test-declarations/pre-test-declarations.module';
+import { Candidate } from '../../../shared/models/DJournal';
+import { Mock } from 'typemoq';
 
 describe('WaitingRoomPage', () => {
   let fixture: ComponentFixture<WaitingRoomPage>;
   let component: WaitingRoomPage;
   let store$: Store<StoreModel>;
   let storeDispatchSpy: jasmine.Spy;
 
+  const moqCandidate = Mock.ofType<Candidate>();
+  moqCandidate.setup((x: Candidate) => x.driverNumber).returns(() => '123');
+  moqCandidate.setup((x: Candidate) => x.candidateName).returns(() => ({ firstName: 'Joe', lastName: 'Bloggs' }));
```

Comment:
These mocks are setup and put in the store but aren't used to test anything? Are they needed?

Reply:
Without setting up these mocks for the `candidate` StoreModule, the DOM tests for this page fail, because the template tries to navigate properties on the candidate retrieved from the store.